LoklakDotNet/Loklak.cs

```diff
@@ -87,8 +90,11 @@ public Loklak(String apiUrl = "http://loklak.org/api/")
         /// <returns></returns>
         public async Task<string> user(string screen_name, int follower_count=0, int following_count=0)
         {
-            var qs = "screen_name=" + screen_name + "&followers=" + follower_count.ToString() + "&following=" + following_count.ToString();
-            System.Diagnostics.Debug.WriteLine(qs);
+            Dictionary<string,string> qs = new Dictionary<string,string>();
+
+            qs.Add("screen_name",screen_name);
+            qs.Add("followers",follower_count.ToString());
```

Comment:
Change this to be `, follower_count. ...` need the spacing after the `,` so that it looks more pleasing to read this.